catapult-sdk/src/plugins/mosaic.js

```diff
@@ -54,7 +54,7 @@ const mosaicPlugin = {
 
 			height: ModelType.uint64,
 			owner: ModelType.binary,
-			properties: { type: ModelType.array, schemaName: ModelType.uint64 }
+			properties: { type: ModelType.array, schemaName: 'mosaicDefinition.mosaicProperty' }
```

Comment:
we will need to think something in the near future to add tests to those cases, i just saw the project does not test this in any other situation